Mathlib/Logic/Embedding/Basic.lean

```diff
@@ -329,11 +329,33 @@ theorem arrowCongrRight_apply {α : Sort u} {β : Sort v} {γ : Sort w} (e : α
 /-- An embedding `e : α ↪ β` defines an embedding `(α → γ) ↪ (β → γ)` for any inhabited type `γ`.
 This embedding sends each `f : α → γ` to a function `g : β → γ` such that `g ∘ e = f` and
 `g y = default` whenever `y ∉ range e`. -/
+@[simps]
 noncomputable def arrowCongrLeft {α : Sort u} {β : Sort v} {γ : Sort w} [Inhabited γ] (e : α ↪ β) :
     (α → γ) ↪ β → γ :=
   ⟨fun f => extend e f default, fun f₁ f₂ h =>
     funext fun x => by simpa only [e.injective.extend_apply] using congr_fun h (e x)⟩
 
+theorem arrowCongrLeft_apply_equiv {α : Sort u} {β : Sort v} {γ : Sort w} [Inhabited γ] (e : α ≃ β)
+    (f : α → γ) (b : β) :
+    (arrowCongrLeft e f) b = f (e.symm b) := by
+  simp
```

Comment:
I'd argue for dropping this one; it's probed by `simp`, and probably any caller should be using `Equiv.arrowCongr` instead if they have an equiv.